app/components/first/first.html

```diff
@@ -11,7 +11,7 @@
 			</div>
 			<div ng-class="first.discoveredHosts.length > 6 ? 'div-with-scroll': ''">
```

Comment:
what happens if there are 5 hosts and screen size is small(ie. 1 host in a row)? Will there be a scroll?